cumulusci/cli/tests/test_cci.py

```diff
@@ -893,19 +893,27 @@ def test_org_connect(self, oauth):
             status=200,
         )
         responses.add("GET", "https://instance/services/data", json=[{"version": 45.0}])
-        run_click_command(
-            cci.org_connect,
-            runtime=runtime,
-            org_name="test",
-            sandbox=False,
-            login_url="https://login.salesforce.com",
-            default=True,
-            global_org=False,
-        )
+        org_config = False
+
+        def wrapper(*args, **kwargs):
+            nonlocal org_config
+            org_config = OrgConfig(*args, **kwargs)
+            org_config.save = mock.Mock()
+            return org_config
+
+        with mock.patch("cumulusci.cli.cci.OrgConfig", wrapper):
+            run_click_command(
+                cci.org_connect,
+                runtime=runtime,
+                org_name="test",
+                sandbox=False,
+                login_url="https://login.salesforce.com",
+                default=True,
+                global_org=False,
+            )
 
         runtime.check_org_overwrite.assert_called_once()
-        runtime.keychain.set_org.assert_called_once()
-        org_config = runtime.keychain.set_org.call_args[0][0]
```

Comment:
Was there something wrong with this way of doing it? The new code is more complicated and doesn't actually assert that the org was stored in the expected keychain. (But I suppose that is probably covered by the tests of OrgConfig itself...)